EIPS/2020 Block Reward Reduction.md

```diff
@@ -0,0 +1,87 @@
+---
+eip: <to be assigned>
+title: 2020 Block Reward Reduction to 0.5 ETH
+author: John Lilic @jlilic twitter.com/johnlilic johnlilicEIP@gmail.com, Jerome de Tychey @jdetychey twitter.com/jdetychey,  Others
+discussions-to: https://ethereum-magicians.org/t/eip-2878-block-reward-reduction-to-0-5-eth/
+status: Draft
+type: Standards Track
+category Core
+created: 2020-08-11
+---
+
+
+## Simple Summary
+
+This EIP will reduce the block reward paid to proof-of-work validators.  Reducing the block reward will maintain the status quo of periodic community-activated block reward reductions.  
+
+## Abstract
```

Comment:
IMO, this is too sensational and should focus on the technical issue--over-paying for security.

Reply:
Can you be more specific?  We state that the goal is to stop over-paying for security, with only one reference to other blockchains.   Do you suggest not justifying or explaining why over-paying is bad in the abstract?

Reply:
EIP abstracts are generally summaries of the technical changes an EIP is proposing, with minimal motivation / rationale, since there are separate sections for that information.